pkg/actuators/machineset/profiles.go

```diff
@@ -0,0 +1,353 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machineset
+
+// Profile is spec of IBM Cloud instance
+type Profile struct {
+	Profile       string
+	VCPU          int64
+	MemoryGb      int64
+	BandwidthGbps int64
+}
+
+// Profiles is a map of IBM Cloud instance resources
+var Profiles = map[string]*Profile{
+	"bx2-2x8": {
```

Comment:
How was this list generated?

Reply:
Manually.

...I'm thinking about updating this so we can dynamically pull the list of profiles and validate.